From 21df87eec8fc11b305217811a5293f7dfaa38cd9 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Sun, 8 Dec 2024 12:35:20 +0300
Subject: [PATCH 05/10] base: providing a choice of additional vndk to build
 image with ext4
 
Also add VNDK 30 for all build types
---
 base.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base.mk b/base.mk
index f6f3982..2de7b50 100644
--- a/base.mk
+++ b/base.mk
@@ -255,4 +255,4 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/window_extensions.mk)
 PRODUCT_PRODUCT_PROPERTIES += \
     persist.settings.large_screen_opt.enabled=true
 
-PRODUCT_EXTRA_VNDK_VERSIONS += 28 29
+PRODUCT_EXTRA_VNDK_VERSIONS += 30
-- 
2.54.0

